test/build/runner.js

```diff
@@ -66,16 +68,8 @@ for (const testName of readdirSync(__dirname)) {
       }
 
       return assert.strictEqual(
-        stripWhitespace(
-          stripChunkHash(
-            stripRev(readFileSync(path.join(entry.path1, entry.name1), {encoding: 'utf8'})),
-          ),
-        ),
-        stripWhitespace(
-          stripChunkHash(
-            stripRev(readFileSync(path.join(entry.path2, entry.name2), {encoding: 'utf8'})),
-          ),
-        ),
+        format(readFileSync(path.join(entry.path1, entry.name1), {encoding: 'utf8'})),
+        format(readFileSync(path.join(entry.path2, entry.name2), {encoding: 'utf8'})),
```

Comment:
Opinionated test runner cleanup 🤷 Ran across this while making sure there wasn’t any Windows-specific fixes in the runner itself